lms/djangoapps/lti_provider/views.py

```diff
@@ -48,12 +48,22 @@ def lti_launch(request, course_id, usage_id):
           pair
     """
     if not settings.FEATURES['ENABLE_LTI_PROVIDER']:
+        log.error(
+            'LTI Provider not enabled in settings for course %s and usage_id %s',
+            course_id,
+            usage_id,
```

Comment:
What is usage ID here? is it client ID for LTI config? If yes, can we skip its logging?